src/amber/environment/settings.cr

```diff
@@ -10,16 +10,17 @@ module Amber::Environment
       skip: Array(String?),
       context: Array(String?))
 
+    alias Value = String | Int32 | Bool | Nil
```

Comment:
`Value` is the name of an abstract struct: https://crystal-lang.org/api/0.24.1/Value.html

Since this is namespaced I don't think it will technically matter, but I think calling it something more specific will help differentiate it.

`SettingValue` may not be the most elegant but I think it is semantically correct.

Reply:
It's borderline [Smurf code](https://blog.codinghorror.com/new-programming-jargon/) but I think I agree. Thanks @marksiemers